service-api/module/Application/src/Helpers/CaseOutcomeCalculator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers;
+
+use Application\Model\Entity\CaseData;
+use Application\Fixtures\DataWriteHandler;
+use Application\Sirius\EventSender;
+use Psr\Log\LoggerInterface;
+
+class CaseOutcomeCalculator
+{
+    public function __construct(
+        private readonly DataWriteHandler $dataHandler,
+        private readonly LoggerInterface $logger,
+        private readonly EventSender $eventSender,
+    ) {
+    }
+
+    public function updateSendIdentityCheck(CaseData $caseData, string $time): void
```

Comment:
```suggestion
    /**
     * @param CaseData $caseData
     * @param string $time
     * @return void
     */
    public function updateSendIdentityCheck(CaseData $caseData, string $time): void
```

Reply:
We generally only add docblocks to expand on types where PHP doesn't do specifics, e.g.:
```
/**
 * @return array{success: bool}
 */
public function saveAnswer(string $answer): array
```
Arrays and generics are the most common case, now that PHP's native typing is pretty well developed

Reply:
Understood